art/__init__.py

```diff
@@ -4,11 +4,11 @@
 import logging.config
 
 # Project Imports
-from art import attacks
+# from art import attacks
 from art import defences
-from art import estimators
-from art import metrics
-from art import wrappers
```

Comment:
Same behavior as for `metrics`. (Note: estimators and attacks work fine when removed)

Reply:
No, this was not intended and needs to be fixed.